<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The Film App</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1001</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QWidget" name="mainMenu">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="admin">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="user">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>150</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="adminMode">
     <widget class="QListView" name="filmList">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>900</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="titleAdmin">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Admin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>50</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="edit">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>500</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="userMode">
     <widget class="QLabel" name="titleUser">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>40</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
     <widget class="QListView" name="watchList">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>900</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="discover">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>50</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Discover</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_2">
      <property name="geometry">
       <rect>
        <x>725</x>
        <y>50</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remove">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>500</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
